ebpf-io/ebpf.io-website
# feat: develop labs page

**This PR:**
- Adds a page with labs;
- Adds labs to the header and footer.

TODO before merge:
- [x] Remove test labs.

**Steps to test**

- Open the [Labs page](https://deploy-preview-438--ebpfio.netlify.app/labs/)
- Confirm that everything looks and works as expected
